src/CollAction/ValidationAttributes/FileTypeAttribute.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.IO;
+using System.Linq;
+
+namespace CollAction.ValidationAttributes
+{
+    public class FileTypeAttribute : ValidationAttribute
```

Comment:
This seems like a decent idea, but is unused. Don't check in classes you don't use.